src/TicketService.java

```diff
@@ -12,6 +13,13 @@ public void addTicket(Ticket ticket) {
         tickets.add(ticket);
     }
 
+    public List<Ticket> getTicketsByStadiumSector(char stadiumSector) {
+        return tickets.stream()
+                .filter(ticket -> ticket.getStadiumSector() == stadiumSector)
+                .collect(Collectors.toList());
```

Comment:
.collect(Collectors.toList()); could be replaced with just toList()

Reply:
Thank you so much! Well noted!